CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 ## DFX
 
+### feat: deploy NNS canisters
+
+A developer is now able to install NNS canisters, including back end canisters such as ledger and governance, and front end canisters such as nns-dapp and internet-identity, on their local DFX server.  Usage:
+```
+dfx start --clean --background
```

Comment:
Does this need to be as follows for the time being?
```suggestion
dfx start --clean --background --host 127.0.0.1:8080
```

Reply:
Is that not the default?  The default works fine, indeed the canisters are compiled for the default.

Reply:
The default is either 4943 (for the shared local network) or 8000 (for a project-specific network)